### DISCUSSION TITLE
Test fail on FreeBSD

### DISCUSSION THREAD

**Discussion by mekanix** [florimondmanca/asgi-lifespan#54] [Q&A]
I'm having trouble with tests. I attached the log of "make test" command. Could it be that pytest-4 is too old and it requires pytest-6 (upgrade is in progress on FreeBSD bugzilla)?
[asgi-lifespan.log](https://github.com/florimondmanca/asgi-lifespan/files/7807135/asgi-lifespan.log)

**Comment by florimondmanca** [comment]
@mekanix Hello!

Just seeing this now.

Yes, I assume pytest 4.x is too old. FYI, currently we're testing on pytest 7.x.

**Comment by mekanix** [comment]
[The PR](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=257334) is ready and when it gets accepted it will be part of FreeBSD ports. Thank you!
